src/app/components/util/chpl-sortable-headers.jsx

```diff
@@ -20,6 +20,13 @@ const useStyles = makeStyles(() => ({
     top: 20,
     width: 1,
   },
+  stickyColumn: {
+    position: 'sticky',
+    left: 0,
+    boxShadow: 'rgba(149, 157, 165, 0.1) 0 4px 8px',
+    backgroundColor: '#ffffff',
+    zIndex:'99',
```

Comment:
```suggestion
    zIndex: '99',
```

Needs a space here for linting purposes